Docs update (#4361)
rotki/rotki: docs/usage_guide.rst

Cointracking.info
====================

You can also import data from `cointracking.info <https://cointracking.info/>`_ into rotki by clicking on "Import Data" on the left sidebar and then following the instructions.

rotki can import any trade CSV data exported from cointracking.info. But in general it's not recommended to utilize cointracking as their exported data are missing a lot of information.

.. image:: images/sc_data_import.png
   :alt: Importing data from cointracking.info
   :align: center

Binance.com
=============

You can import data from `binance.com <https://binance.com/>`_ into rotki by clicking on "Import Data" on the left sidebar and then following the instructions.

> rotki can import a CSV data exported from Binance. But due to Binance's csv format some data may be not importable. You will see warnings if this will happen.

Reviewer: ```suggestion
rotki can import a CSV data file exported from Binance. But due to Binance's csv format some data may be not importable. You will see warnings if this happens.
```

Author: Alright

> 

Reviewer 2: I would add here that the import functionality can import more things than the API provides

Author: Aha, yep, sounds cool

Reviewer: yeah please do add this distinction

Author: Sure, added!

Cointracking.info
====================

You can also import data from `cointracking.info <https://cointracking.info/>`_ into rotki by clicking on "Import Data" on the left sidebar and then following the instructions.

rotki can import any trade CSV data exported from cointracking.info. But in general it's not recommended to utilize cointracking as their exported data are missing a lot of information.

.. image:: images/sc_data_import.png
   :alt: Importing data from cointracking.info
   :align: center

Binance.com
=============

You can import data from `binance.com <https://binance.com/>`_ into rotki by clicking on "Import Data" on the left sidebar and then following the instructions.

> By importing a csv file you are able to import more than with API. Particularly: Trades, Deposits and Withdrawals, Small assets exchange BNB, ETH 2.0 Staking and ETH 2.0 Staking Rewards, Launchpool Interests, POS savings interest, POS savings purchase, POS savings redemption.

Reviewer: ```suggestion
By importing a csv file you are able to import more than with the API. Particularly: Trades, Deposits and Withdrawals, Small assets exchange BNB, ETH 2.0 Staking and ETH 2.0 Staking Rewards, Launchpool Interests, POS savings interest, POS savings purchase, POS savings redemption.
```

> rotki can import a CSV data file exported from Binance. But due to Binance's csv format some data may be not importable. You will see warnings if this happens.

Reviewer: This should be above, and the "By importing" should be last.

   :align: center

ENS names resolution
=====================

> rotki automatically resolves ens name for each your ethereum account. If there is an ens name for an account, this name will be shown instead of raw blockchain address. You can find the blockchain address by hovering the ens name.

Reviewer 2: Isn't it better?
```suggestion
rotki automatically resolves ens name for each of your ethereum accounts. If there is an ens name for an account, this name will be shown instead of raw blockchain address across the app. You can find the blockchain address by hovering the mouse over the ens name.
```

Author: Sounds cool, will change it, thks

Reviewer: Not exactly. It's all cool but please specify: 

`If there is an ens name for an account` -> `If there is a primary ens name specified for an account`

This is because if you don't set the primary ens name for your address it won't be shown. 

P.S.: Do you have an ENS @nebolax? you should get one!

Author: Oh yeah, thank you Lefteris!

Filtering deposits & withdrawals
================================

> You can filter your deposits and withdrawals in the same say you can filter your trades.

Reviewer: ```suggestion
You can filter your deposits and withdrawals in the same way you can filter your trades.
```

Figure out which asset caused the price not found. Then check the historical price caches and make sure you have the historical price cache for that asset pair created. For example if you are creating a GBP profit/loss report and the asset is GNO then make sure to create the GNO -> GBP historical price cache. See :ref:`manage-historical-price-cache` on how to do it.

> Statistics

Author: Renaming `Analytics` to `Statistics` since the sidebar option is called `Statistics` atm

Reviewer: Hmmm ... here we should be consistent between both the app and the website. In the site we call them "Analytics" or "Graphs": https://rotki.com/products/

![2022-05-26_22-26](https://user-images.githubusercontent.com/1658405/170574435-de0acb68-ced7-47d9-9a37-988fc18b977d.png)

cc @kelsos what do you think?

Author: As discussed at standup, will keep it `Statistics`.
